### ISSUE TITLE
intellij-scala/build.sbt:180: error: missing parameter type

### ISSUE THREAD

**Issue by codygman** [JetBrains/intellij-scala#129]
```shell
cody@cody-G46VW:~/IdeaProjects/intellij-scala$ sbt install downloadIdea
[info] Loading global plugins from /home/cody/.sbt/0.13/plugins
[info] Loading project definition from /home/cody/IdeaProjects/intellij-scala/project
/home/cody/IdeaProjects/intellij-scala/build.sbt:180: error: missing parameter type
  lazy val resolved = (
           ^
[error] Type error in expression
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? 
cody@cody-G46VW:~/IdeaProjects/intellij-scala$ scala
Welcome to Scala version 2.11.0 (OpenJDK 64-Bit Server VM, Java 1.7.0_75).
Type in expressions to have them evaluated.
Type :help for more information.
cody@cody-G46VW:~/IdeaProjects/intellij-scala$ sbt --version
sbt launcher version 0.13.5

```

**Comment by dedoz** [comment]
Does it also happen when you try a different sbt launcher version, 0.13.7 for example?

**Comment by dedoz** [comment]
Anyway, try removing lazy modifier, it should fix the problem.
This is weird because it looks like an old sbt bug which should have been fixed in 0.13.2.

**Comment by nkly** [comment]
This issue is obsolete since some time ago plugin's build was refactored. Closing.
